modules/models/src/main/java/org/eclipse/xpanse/modules/models/service/DeployResource.java

```diff
@@ -19,18 +22,26 @@ public class DeployResource {
     /**
      * The ID of the deployed resource.
      */
+    @NotBlank
```

Comment:
for all strings, please also add `@notNull`

Reply:
Ok.